bitnami/wordpress/values.yaml

```diff
@@ -1217,3 +1217,93 @@ externalCache:
   ## @param externalCache.port External cache server port
   ##
   port: 11211
+
+## @section CronJob parameters
+##
+
+## Configure cron jobs used to perform periodic tasks
+##
+cronJobs:
+  scheduler:
```

Comment:
It seems there's only one cronjob you're actually defining. Do you expect to add support for more? Isn't this sth that can be already covered with `extraDeploy` parameter?